Doc/pages/gui.rst

```diff
@@ -1,16 +1,333 @@
 
-Using MDANSE Graphical User Interface
+.. _section-7:
```

Comment:
This label should, if possible, refer to the contents of the text and not number of the section.